samples/nametranslation.d.ts.scala

```diff
@@ -0,0 +1,16 @@
+
+import scala.scalajs.js
+import js.annotation._
+import js.|
+
+package nametranslation {
+
+@js.native
+@JSGlobalScope
+object Nametranslation extends js.Object {
+  val uint32Array: js.typedarray.Uint32Array = js.native
+  val readonlyArray: js.Array = js.native
+  val promiseLike: js.Thenable = js.native
```

Comment:
The two lines above do not compile, because `js.Array` and `js.Thenable` need a type parameter.